src/config.cc

```diff
@@ -33,6 +35,7 @@ void PrepareCommanLineParser(cmdline::parser& parser)
     parser.add<std::string>(config::Format,    'f', "output format of the AST (yaml|json)", false, "yaml", cmdline::oneof<std::string>("yaml", "json"));
     parser.add<std::string>(config::Type,      't', "type of the AST (refract|ast)", false, "refract", cmdline::oneof<std::string>("refract", "ast"));
     parser.add<std::string>(config::Sourcemap, 's', "export sourcemap AST into file", false);
+    parser.add(config::RefractSourceMap,       'r', "produce sourcemap for refract AST");
```

Comment:
we disscus it with @zdne and decide, there should be new commandline option.
while you produce sourcemap for refract it is generated directly into refract ouput, there is no extra file with sourcemap. So it is strange to have option `-s` with `sourcemap`. 
Yes, there is possibility to do not generate source map, but it is non consistent behavior. 
Because behavior of one option change behavior of other option. 

So we decide for additional switch.